test/addons-napi/test_general/testInstanceOf.js

```diff
@@ -62,7 +62,7 @@ if (typeof Symbol !== 'undefined' && 'hasInstance' in Symbol &&
 
   function compareToNative(theObject, theConstructor) {
     assert.strictEqual(addon.doInstanceOf(theObject, theConstructor),
```

Comment:
If we have to split parameters into several lines, I'd like to let first parameter use a new line also.

eg.

```javascript
assert.strictEqual(
  addon.doInstanceOf(...),
  (theObject instanceof theConstructor));
```